src/Explorer/Explorer.tsx

```diff
@@ -428,11 +440,14 @@ export default class Explorer {
       );
       return;
     }
+    const dialogContent = NotebookUtil.isPhoenixEnabled()
+      ? "Notebooks saved in temporary workspace will also be lost. Proceed anyway?"
```

Comment:
@deborahc / @swviswan,  Please suggest text here. This is a pop-up text that comes on click of Reset workspace button.